assets/app/index.rb

```diff
@@ -5,6 +5,32 @@ def render
     css = <<~CSS
       * { font-family: 'Inconsolata', monospace; }
 
+      @media (prefers-color-scheme: dark) {
+        body {
+          background-color: black;
+          color: #eee;
+        }
+        .nav #logo--dark path {
```

Comment:
can we get rid of the doubles and call it 
#logo_dark

Reply:
We could, but again BEM naming convention.